catalyst-gateway/bin/src/service/api/documents/mod.rs

```diff
@@ -115,8 +115,8 @@ impl DocumentApi {
         &self, /// The Query Filter Specification
         query: Json<DocumentIndexQueryFilterBody>,
         page: Query<Option<Page>>, limit: Query<Option<Limit>>,
-        /// Authorization required.
-        _auth: CatalystRBACSecurityScheme,
+        /// No Authorization required, but Token permitted.
+        _auth: NoneOrRBAC,
```

Comment:
we don't need auth for this?

Reply:
yes, this endpoint does not require it

Reply:
More importantly, the front end needs it when it might not yet have any registrations